compiler_opt/es/regalloc_trace/regalloc_trace_worker.py

```diff
@@ -44,8 +46,16 @@ class RegallocTraceWorker(worker.Worker):
   segments.
   """
 
-  def __init__(self, clang_path: str, basic_block_trace_model_path: str,
-               thread_count: int, corpus_path: str):
+  def _setup_base_policy(self):
+    self._tf_base_temp_dir = tempfile.mkdtemp()
+    policy = policy_utils.create_actor_policy()
+    saver = policy_saver.PolicySaver({"policy": policy})
+    saver.save(self._tf_base_temp_dir)
+    self._tf_base_policy_path = os.path.join(self._tf_base_temp_dir, "policy")
+
+  def __init__(self, gin_config: str, clang_path: str,
```

Comment:
can you force these to be passed by-name, I couldn't for the life of me figure out what `""` meant in tests, for the first parameter.

Reply:
Done.